gufe/tests/test_ligand_network.py

```diff
@@ -47,9 +47,9 @@ def mols():
 @pytest.fixture
 def std_edges(mols):
     mol1, mol2, mol3 = mols
-    edge12 = LigandAtomMapping(mol1, mol2, {0: 0, 1: 1})
-    edge23 = LigandAtomMapping(mol2, mol3, {0: 0})
-    edge13 = LigandAtomMapping(mol1, mol3, {0: 0, 2: 1})
+    edge12 = LigandAtomMapping(mol1, mol2, {0: 0, 1: 1}, {"score": 0.0})
```

Comment:
Could you test other types of annotations too? For example, what if you passed a united quantity - from the code, it looks like that might fail ungracefully.

Reply:
Good point we could add support for all of our JSON codec entries here, it might be worth considering transitioning to the new `to_json` and `from_json` methods to serialise these networks generally.

Reply:
For now unfortunately we do have to fix the graphml serialisation because it's what we've been getting out users to use, so for the rest of 1.x we have an obligation to keep this going. 

Probably just trying to see if we can use the standard JSON decoder/encoder on the annotations might be worth it.